laplace/curvature/backpack.py

```diff
@@ -85,6 +85,34 @@ def gradients(self, x, y):
                         for p in self._model.parameters()], dim=1)
         return Gs, loss
 
+    def gp_quantities(self, X, y, sigma_factor):
```

Comment:
Could this be a part of `CurvatureInterface`, too? Then `ASDL` is supported right away if I am not mistaken.

Reply:
moved it to `GGNInterface`. tested `FunctionalLaplace` with `AsdlGGN` backend and it's working, see test `test_gp_equivalence_classification` in [test_functional_laplace.py](https://github.com/metodj/Laplace/blob/functional-laplace/tests/test_functional_laplace.py)